google-cloud-storage/lib/google/cloud/storage/policy.rb

```diff
@@ -162,43 +186,76 @@ def role role_name
         end
 
         ##
-        # Returns a deep copy of the policy.
+        # Returns a deep copy of the policy. The service object referenced by
```

Comment:
This note about the service object is going to be confusing. The fact that the Policy object holds a reference to a Service object is outside of the public API. I think this change to the documentation should be reverted.